backends/dpdk/spec.cpp

```diff
@@ -455,16 +452,17 @@ std::ostream& IR::DpdkLearner::toSpec(std::ostream& out) const {
     }
     out << std::endl;
     if (auto size = properties->getProperty("size")) {
-        out << "\tsize " << DPDK::toStr(size->value) << "" << std::endl;
+        out << "\tsize 0x" << std::hex << DPDK::toStr(size->value) << "" << std::endl;
```

Comment:
Hope std::hex is not required here as the next operand is a string.

Reply:
Yes. Thanks for pointing out. Will update

Reply:
updated